lib/core/form/components/widgets/form.theme.scss

```diff
@@ -27,10 +26,15 @@ ul > li > form-field > .adf-focus {
     }
 
     &-label {
-        color: rgb(186, 186, 186);;
+        color: rgb(186, 186, 186);
     }
 
     &-invalid {
+        .mat-form-field-label,
+        .mat-form-field-required-marker,
+        .mat-input-element ::placeholder {
+            color: #f44336 !important;
```

Comment:
can we use a custom error state matcher instead of overriding the css? More details here https://material.angular.io/components/input/overview

Reply:
color: var(--theme-warn-color);

Reply:
i changed approach as Ardit suggested